var/spack/repos/builtin/packages/py-neurom/package.py

```diff
@@ -32,6 +33,10 @@ class PyNeurom(PythonPackage):
     depends_on('py-scipy@1.2.0:', type='run')
     depends_on('py-plotly@3.6.0:', type='run', when='+plotly')
 
+    # >= 2.0
+    depends_on('py-morphio@3.1.1:', type='run', when='@2.0:')
+    depends_on('py-pandas@1.0.5:', type='run', when='@2.0:')
```

Comment:
I did in this way because it seemed clearer to me to split the dependencies based on the `py-neurom` version.
However, I'll try to merge `py-pandas` so we'll have one line less.